src/ray/protobuf/core_worker.proto

```diff
@@ -71,6 +71,15 @@ message ReturnObject {
   int64 size = 6;
 }
 
+message StealWorkRequest {
+  int64 max_tasks_to_steal = 1;
+}
+
+message StealWorkReply {
+  int64 number_of_tasks_stolen = 1;
```

Comment:
I don't think we need this field since we can just check the size of `tasks_stolen`.